change/@fluentui-react-table-1a06bc91-0c0f-464f-a1c3-4523427d45a1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28940&commitId=8af5f09741638631821ebde23f5934f54e1febc3&env=prod&clientType=FLUENTUI)


  <details><summary><b>Table layout flex - truncate</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - truncate.true.chromium.png|6919| Changed|
|Table layout flex - truncate.false.chromium.png|11832| Changed|
|Table layout flex - truncate.default (disabled).chromium.png|11832| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>